services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
```

Comment:
You can write it easyer: movie=movie_id

Reply:
it raises error: ValueError: Cannot assign "1": "MovieSession.movie" must be a "Movie" instance.

Reply:
Got it. My bad. movie_id=movie_id. Same for cinema_hall. Try it